site/docs/get-started/quickstart.md

```diff
@@ -108,26 +108,46 @@ Run the following to install vault and the plugin in development mode:
     kubectl wait --for=condition=ready --all pod -n fybrik-system --timeout=120s
     ```
 
+## Install data catalog
+
+Fybrik assumes the existence of a data catalog that contains the metadata and connection information for data assets. Fybrik currently supports:
+
+ 1. [OpenMetadata](https://open-metadata.org/): An open-source end-to-end metadata management solution that includes data discovery, governance, data quality, observability, collaboration, and lineage.
+ 2. [katalog](https://fybrik.io/dev/reference/katalog/): a data catalog that is included in Fybrik for evaluation purposes.
+
+In this guide we use the OpenMetadata data catalog.
+
+To use OpenMetadata, you can either use an existing deployment, or run the following commands to deploy OpenMetadata in kubernetes:
+```bash
+curl https://raw.githubusercontent.com/fybrik/fybrik/master/third_party/openmetadata/install_OM.sh | bash -
+```
+
```

Comment:
I think we have to add OMD installation instructions for OpenShift. (I sent them to you)